src/.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": [
+    "@integreat-app/eslint-config",
+    "@integreat-app/eslint-config-react",
+    "@integreat-app/eslint-config-jest",
+    "plugin:flowtype/recommended"
+  ],
+  "plugins": [
+    "flowtype"
+  ],
+  "rules": {
+    "flowtype/require-valid-file-annotation": [
```

Comment:
These rules should be in @integreat-app/eslint-config

Reply:
Sure, these rules, shouldn't be right here.
Take a look at https://issues.integreat-app.de/browse/LINT-4
Do you agree with that?